nodes/sros/configs/grpc_insecure.cfg

```diff
@@ -0,0 +1,5 @@
+configure system grpc admin-state enable
+configure system security user-params local-user user "admin" access grpc true
+configure system grpc allow-unsecure-connection
+configure system grpc gnmi auto-config-save true
+configure system grpc rib-api admin-state enable
```

Comment:
@sacckth can we ensure we enable gnoi for secure/insecure modes as well? I think I don't see it here